### DISCUSSION TITLE
Runtime keybinds in help menu

### DISCUSSION THREAD

**Discussion by ndavd** [jarun/nnn#1759] [Ideas]
Since keybinds are static it should be possible for the help menu to display the key that's being used in a certain action. If I want to change a keybind I can modify `nnn.h`, but the help menu doesn't update accordingly because it's harcoded.

**Comment by ndavd** [comment]
Or is there some specific reason why it hasn't been done? @jarun

**Comment by N-R-K** [comment]
Pressing <kbd>?</kbd> will show you the keybindings.

**Reply by ndavd** [reply]
Maybe I didn't describe the issue well.
What I meant is that if I want to change a keybind I can modify `nnn.h`, but the help menu doesn't update accordingly because it's hardcoded.

**Reply by N-R-K** [reply]
That's because we don't do runtime keybind configuration. So there's not much interest in trying to unhardcode the helpstring.
